app/assets/javascripts/float-kit/addon/services/menu.js

```diff
@@ -66,20 +66,17 @@ export default class Menu extends Service {
     }
 
     if (instance.expanded) {
-      return await this.close(instance);
```

Comment:
what was the problem of this?

Reply:
To me the former syntax suggests that we want to pass through the return value of `this.close()`.

But `this.close()` doesn't return anything (i.e. `Promise<void>`) so I prefer to reflect that at the call site and separate the `return` from `await [call]`. 🙂